app/views/sequences/_form.html.haml

```diff
@@ -32,6 +32,10 @@
     .field
       = f.label :theme_id, "Theme"
       = f.select :theme_id, options_from_collection_for_select(Theme.all, 'id', 'name', @sequence.theme_id), { :include_blank => "None (inherit from project, or site default)" }
```

Comment:
Same approach of disabling the theme for sequences when the AP runtime is selected seems best.

Reply:
@scytacki I made the disable Theme changes as well.